plato/draw/fresnel/Ellipsoids.py

```diff
@@ -0,0 +1,43 @@
+import fresnel
+import itertools
+import numpy as np
+from ... import draw
+from ..internal import ShapeAttribute, ShapeDecorator
+from .FresnelPrimitive import FresnelPrimitive
+
+def fibonacciPositions(n_b, a=.5, b=0.5, c=0.5):
```

Comment:
@klarh We may wish to move this function to a single place. Between this PR and #30 and pythreejs Spheres, it's used 3 times. I would refactor that in #29 after merging #30 and #31.